cardano-crypto-class/src/Cardano/Crypto/VRF/Class.hs

```diff
@@ -207,10 +233,9 @@ decodeCertVRF = do
           expected = fromIntegral (sizeCertVRF (Proxy :: Proxy v))
           actual   = BS.length bs
 
-
 data CertifiedVRF v a
   = CertifiedVRF
-      { certifiedNatural :: Natural
+      { certifiedNatural :: ByteString
```

Comment:
The name of this field is now a little weird... how about `certifiedBytes`?

Reply:
How about `certifiedOutput`?